### DISCUSSION TITLE
Mailgun Error

### DISCUSSION THREAD

**Discussion by fenibofubara** [adonisjs/core#3018] [Help]
Am trying to send mails with mailgun connection on adonis 4.1.0
My configurations are pretty ok, but i get this error:

internal/streams/legacy.js:61
      throw er; // Unhandled stream error in pipe.
      ^

Error: getaddrinfo ENOTFOUND api.us.mailgun.net
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:66:26) {
  errno: 'ENOTFOUND',
  code: 'ENOTFOUND',
  syscall: 'getaddrinfo',
  hostname: 'api.us.mailgun.net'
}

I need answers guys.... Thanks

**Comment by varun-nambiar** [comment]
Where did you get “api.us.mailgun.net” or you guess it?

**Reply by varun-nambiar** [reply]
https://documentation.mailgun.com/en/latest/api-intro.html#base-url

**Reply by fenibofubara** [reply]
MAILGUN_API_REGION=US
MAILGUN_DOMAIN=api.droomwork.io
